pysyncthru/tests/test_structure/server_control.py

```diff
@@ -41,7 +42,7 @@ def stop_server(self):
         if self._server_running:
             self._server.shutdown()
 
-    def start_server(self, timeout=10):
+    def start_server(self, timeout: int = 10) -> None:
```

Comment:
IMHO the type is already hinted by the default value. However explicitely noting it does not hurt here.

Reply:
Nothing in the unhinted signature would prevent it from being e.g `Optional` or a `Union` with some other type.